### ISSUE TITLE
[Feature Request] Report to screen reader users when event recording starts and stops

### ISSUE THREAD

**Issue by codeofdusk** [microsoft/accessibility-insights-windows#1440]
**Is your feature request related to a problem? Please describe.**
Although apparently displayed visually, it is immediately unclear to screen reader users when event recording starts or stops.

**Describe the solution you'd like**
A few possible solutions:
* Raise UIA notification events when event recording starts and stops:
    * Works for both speech and Braille users.
    * May not be viable, as NVDA [ignores notification events from background apps](https://github.com/nvaccess/nvda/blob/master/source/NVDAObjects/UIA/__init__.py#L2037) and event recording is toggled using a global keyboard shortcut pressed while the app under inspection is in focus. I'm not sure if this is common to all screen readers – maybe @josephsl knows why NVDA does this?
* Play a short, distinct sound:
    * Sound should be different for starting and stopping recording.
    * Braille-only users still may not know whether event recording starts and stops (such as deafblind users who can't hear the sound).

**Describe alternatives you've considered**
Keep the behaviour as-is.

**Comment by josephsl** [comment]
Hi, when I worked on IUIAutomation5’s notification event handler, I intended notifications from all apps to be announced by NVDA, but @leonardDeR pointed out that people would be distracted if they hear volume level change notifications while using apps other than File Explorer, hence the compromise was made to announce notifications from focused app. There is (or rather was) a pull request that introduced a setting to let NVDA announce notifications from everywhere, only from focused app, or no announcements. Introducing this setting to NVDA will allow folks to use raise notification function from UIA for various tasks, although keep in mind that there are restrictions placed in places such as Windows Calculator where handling notification event can become a distraction (more so when an event in addition to UIA notification (such as name property change) is also fired by the same object that notified users fraction of a second prior). Thanks.
